### PR TITLE
Gp 3210 use aggregate skip status from cerberus

### PR DESCRIPTION
**Pull request by mlaszloffy**
This won't build correctly until the cerberus 0.2.10 release.

JIRA Ticket: GP-3210

- [X] Updates Changelog
- [] Updates developer documentation

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -3,16 +3,16 @@ FROM debian:bullseye
...
-RUN cd /build && \
+RUN --mount=type=cache,target=/root/.m2 cd /build && \
     tsc -p shesmu-server-ui && \
-    mvn -q -Dsurefire.useFile=false -DskipIT=true clean install && \
+    mvn -Dsurefire.useFile=false -DskipIT=true clean install && \
     mkdir /usr/share/shesmu && \
     cp shesmu-pluginapi/target/shesmu-pluginapi.jar shesmu-server/target/shesmu.jar plugin-*/target/shesmu-plugin-*.jar /usr/share/shesmu
 
-FROM openjdk:16-slim
+FROM openjdk:17
```
Comment (avarsava) [inline Dockerfile:15]: I have yet to verify that shesmu runs correctly with java 17?

Comment (callunity) [reply Dockerfile:15]: We don't use the Dockerfile in our local deploy - I believe it's only used to build a Docker image during the release (and that hasn't been working reliably recently).

**Review by callunity — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (avarsava) [reply Dockerfile:15]: Why update it in this PR?

File: plugin-cerberus/pom.xml [hunk 2]
```diff
@@ -48,7 +48,7 @@
     <dependency>
       <groupId>ca.on.oicr.gsi.cerberus</groupId>
       <artifactId>cerberus-core</artifactId>
-      <version>0.2.9</version>
+      <version>0.2.10-SNAPSHOT</version>
```
Comment (callunity) [inline plugin-cerberus/pom.xml:51]: Cerberus 0.2.10 is now released, so this can be updated

File: Dockerfile (hunk 1 above)
Comment (mlaszloffy) [reply Dockerfile:15]: I was testing with the dockerfile locally, I can split this out into another PR for later.

**Review by avarsava — APPROVED** [review]

**Pull request merged**